Makefile

```diff
@@ -195,15 +195,15 @@ build-images:
 		echo "Ensure you have exported a version to build!";\
 		exit 1
 	fi
-	autonomy deploy build image ${SERVICE_ID} --dependencies || (echo failed && exit 1)
```

Comment:
Several options like this one have been detected by the new tool as not valid. After verifying it in the help section of the CLI, I deleted them, but I would like @8ball030 adn @angrybayblade to double check just in case this now invalidates some commands.